### ISSUE TITLE
Desktop/Panel Widget

### ISSUE THREAD

**Issue by alanbortu** [DoTheEvo/ANGRYsearch#6]
Any plans for a KDE/Gnome 3 widget for the desktop/panel? I think it would be pretty useful to have ANGRYsearch in a form similar to [Milou/Baloo](http://vhanda.in/blog/2014/03/introducing-milou/).

Example screenshot:
![Milou](http://vhanda.in/images/milou-preview.png)

**Comment by DoTheEvo** [comment]
Not really, I know nothing about kde/gnome widgets, I am just learning PyQt as I go...

**Comment by alanbortu** [comment]
Hmm okay, it would be interesting to have an applet like that at least in KDE since you wouldnt have to do too much porting..
